### PR TITLE
Adjust-the-privileges-options 

### PR DESCRIPTION
**Pull request by LinconOssa**
The change requested for each company is left only shown for Users, linix use cases, Roles and Positions. for online systems and Users, linix use cases, for others

### CODE REVIEW

**Review by ANGIECATHERINEP — APPROVED** [review]
The request was made correctly

File: src/pages/privileges/outlets/options/index.tsx [hunk 1]
```diff
@@ -1,13 +1,25 @@
 import { PrivilegesOptionsUI } from "./interface";
```
Comment (carin91) [inline src/pages/privileges/outlets/options/index.tsx:1]: adjust the imports

File: src/pages/privileges/outlets/options/index.tsx [hunk 2]
```diff
@@ -1,13 +1,25 @@
...
 import { privilegeOptionsConfig } from "./config/privileges.config";
-import { appsConfig } from "@components/layout/AppPage/config/apps.config";
-
+import { useContext } from "react";
+import { AppContext } from "@src/context/AppContext";
```
Comment (carin91) [inline src/pages/privileges/outlets/options/index.tsx:4]: Use the appropriate alias.

Comment (LinconOssa) [reply src/pages/privileges/outlets/options/index.tsx]: resolved

File: src/pages/privileges/outlets/options/index.tsx (hunk 1 above)
Comment (LinconOssa) [reply src/pages/privileges/outlets/options/index.tsx]: resolved

**Review by carin91 — APPROVED** [review]

**Pull request merged**